mods/persistence/modules/world_save/serializers/sql_serializer.dm

```diff
@@ -146,7 +159,7 @@
 	inserts_since_commit++
 	thing_map["\ref[object]"] = p_i
 
-	for(var/V in object.get_saved_vars())
+	for(var/V in get_saved_variables_for(object.type))
 		if(!(V in object.vars))
```

Comment:
As per our discussion on discord, this needs to be removed so that the save doesn't slow to a crawl

Reply:
Alright I guess I'll make the change myself then.